buildtools/cli.go

```diff
@@ -1074,3 +1083,15 @@ func getTwineConfigPath() (configFilePath string, err error) {
 	}
 	return "", errorutils.CheckErrorf(getMissingConfigErrMsg("twine", "pip-config OR pipenv-config"))
 }
+
+// Extracts the boolean flag from the args and removes it from the args.
+func extractBoolFlagFromArgs(filteredNugetArgs *[]string, flagName string) (value bool, err error) {
```

Comment:
the func name is generic but it is a specific func for allowInsecureConnection flag